src/Rings/mpoly-affine-algebras.jl

```diff
@@ -250,7 +248,7 @@ julia> degree(A)
 """
 function degree(A::MPolyQuo)
    if iszero(A.I)
-       return 1
+       return ZZ(1)
```

Comment:
Perhaps we think the degree in "realistic" (?? whatever that means?) examples can never overflow an `Int` -- then we could change this back here, but also `degree` for `HilbertData`, called below, ought to be adjusted?

Reply:
Let me put it this way: If degree would overflow  an int64, then the example would be so large that we would not stand a chance to even reach this point, because something on the way (groebner-basis) would have exhausted our memory anyway.
 (Do not be misled by the multiplication with the d! -- this is not a hack, but a knowledgable and well-controlled canceling of the denomiator already present in mathematical textbooks of the subject.)